packages/ui/cypress/support/commands.ts

```diff
@@ -55,7 +55,7 @@ Cypress.Commands.add('initExtension', (accounts: InjectedAccountWitMnemonic[]) =
 })
 
 Cypress.Commands.add('getAuthRequests', () => {
-  return cy.wrap(extension.getAuthRequests())
+  return cy.wait(500).then(() => cy.wrap(extension.getAuthRequests()))
```

Comment:
We can use waitUntil especially when we show the loader. It's hard to guess how much time to really need, so the test might be unstable.
The example of usage
```
 cy.waitUntil(function() {
  return cy.get('loader').should('not.exist');
 })

Reply:
There is a library for that method https://www.npmjs.com/package/cypress-wait-until

It's a missing method in the cypress, but it's inspired by "Testing library" (https://testing-library.com/docs/guide-disappearance/#waiting-for-disappearance).

Reply:
I really dislike `waits`, and I'll fight them all as much as I can. But in this case, I don't think we have any other solution, because the actual firing of the request is itself delayed. the loader appears right away (after this PR) but the firing of the request for accounts is done 500ms after:
https://github.com/ChainSafe/Multix/blob/eac29c0b55c704a88ace236e5ad98af02f1cea34/packages/ui/src/contexts/AccountsContext.tsx#L136-L138

Reply:
I'll check if waitUntil can help here